### PR TITLE
Initial auth

### PR DESCRIPTION
**Pull request by Deseteral**

### CODE REVIEW

File: service/src/main/kotlin/xyz/lebkuchenfm/Application.kt [hunk 1]
```diff
@@ -56,20 +70,63 @@ fun Application.module() {
...
     val discordClient = DiscordClient(environment.config, commandExecutorService)
-
     launch { discordClient.start() }
 
     install(ContentNegotiation) {
         json()
     }
 
+    install(Sessions) {
+        val days30 = 30 * 24 * 60 * 60 * 1000L
```
Comment (gorzalczany) [inline service/src/main/kotlin/xyz/lebkuchenfm/Application.kt:80]: thats in millisecond but variable below is `maxAgeInSeconds`

File: service/src/main/kotlin/xyz/lebkuchenfm/api/auth/AuthEndpoint.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import io.ktor.server.sessions.get
+import io.ktor.server.sessions.sessions
+import io.ktor.server.sessions.set
+import kotlinx.serialization.Serializable
+import xyz.lebkuchenfm.domain.auth.UserSession
+
+fun Route.authRouting() {
+    route("/auth") {
+        get {
+            val session = call.sessions.get<UserSession>()!!
```
Comment (gorzalczany) [inline service/src/main/kotlin/xyz/lebkuchenfm/api/auth/AuthEndpoint.kt:21]: force?

**Review by gorzalczany — CHANGES_REQUESTED** [review]

File: service/src/main/kotlin/xyz/lebkuchenfm/api/auth/AuthEndpoint.kt (hunk 2 above)
Comment (Deseteral) [reply service/src/main/kotlin/xyz/lebkuchenfm/api/auth/AuthEndpoint.kt:21]: Yes. Technically it should never be null. So at least while we're in development mode I would like it to throw Null Pointer Exception so we can debug the case when it is.

File: service/src/main/kotlin/xyz/lebkuchenfm/Application.kt (hunk 1 above)
Comment (Deseteral) [reply service/src/main/kotlin/xyz/lebkuchenfm/Application.kt:80]: You're absolutely right!

**Review by gorzalczany — APPROVED** [review]

**Pull request merged**